cob_twist_controller/include/cob_twist_controller/augmented_solver_data_types.h

```diff
@@ -0,0 +1,68 @@
+/*!
+ *****************************************************************
+ * \file
+ *
+ * \note
+ *   Copyright (c) 2015 \n
+ *   Fraunhofer Institute for Manufacturing Engineering
+ *   and Automation (IPA) \n\n
+ *
+ *****************************************************************
+ *
+ * \note
+ *   Project name: care-o-bot
+ * \note
+ *   ROS stack name: cob_control
+ * \note
+ *   ROS package name: cob_twist_controller
+ *
+ * \author
+ *   Author: Marco Bezzon, email: Marco.Bezzon@ipa.fraunhofer.de
+ *
+ * \date Date of creation: March, 2015
+ *
+ * \brief
+ *   Different data types for AugmentedSolver to be used in other modules
+ *
+ ****************************************************************/
+
+#ifndef AUGMENTED_SOLVER_DATA_TYPES_H_
+#define AUGMENTED_SOLVER_DATA_TYPES_H_
```

Comment:
this classes might need a revision after changes being made due to my comments in the cfg file...